libs/langchain/langchain/callbacks/manager.py

```diff
@@ -233,13 +236,17 @@ def trace_as_chain_group(
             Defaults to None.
 
     Returns:
-        CallbackManager: The callback manager for the chain group.
+        CallbackManagerForChainGroup: The callback manager for the chain group.
 
     Example:
-        >>> with trace_as_chain_group("group_name") as manager:
-        ...     # Use the callback manager for the chain group
-        ...     llm.predict("Foo", callbacks=manager)
-    """
+        .. code-block:: python
+
+            llm_input = "Foo"
+            with trace_as_chain_group("group_name", inputs={"input": llm_input}) as manager:
+                # Use the callback manager for the chain group
+                res = llm.predict(llm_input, callbacks=manager)
+                manager.on_group_end({"output": res})
```

Comment:
why not use on_chain_end ?

Reply:
Bad design most likely. Can't think of a reason to not use it right now